common/primitives/src/parquet/temporal.rs

```diff
@@ -0,0 +1,82 @@
+use codec::{Decode, Encode, MaxEncodedLen};
+use scale_info::TypeInfo;
+use serde::{Deserialize, Serialize};
+use sp_std::prelude::*;
+
+/// Parquet temporal types: https://github.com/apache/parquet-format/blob/master/LogicalTypes.md
+#[derive(
+	Clone, Encode, Decode, PartialEq, Debug, TypeInfo, Eq, MaxEncodedLen, Serialize, Deserialize,
+)]
+pub enum ParquetTemporalType {
+	/// Parquet dates
+	Date,
+	/// Parquet intervals
+	Interval,
```

Comment:
for documentation here maybe include a bit more information from their readMe
annotate a fixed_len_byte_array of length 12.